<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>App</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
border-top-color: rgb(225,225,225);
border-left-color: rgb(225,225,225);
border-right-color: rgb(179,176,176);
border-bottom-color: rgb(179,176,176);
border-bottom-width: 2px;
border-top-width: 1px;
border-left-width: 1px;
border-right-width: 2px;
border-style: solid;
background-color: rgb(225,225,225);
}



QLineEdit{
border-top-color: rgb(225,225,225);
border-left-color: rgb(225,225,225);
border-right-color: rgb(190,188,188);
border-bottom-color: rgb(190,188,188);
border-bottom-width: 2px;
border-top-width: 1px;
border-left-width: 1px;
border-right-width: 2px;
border-style: solid;
background-color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Load files&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="meteoButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Meteo File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tracesButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Module File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>20</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>280</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Select module&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="moduleBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Module 1</string>
    </property>
    <item>
     <property name="text">
      <string>Module 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Module 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Module 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Module 4</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="filterButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>610</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="endDateTime">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>2</month>
      <day>8</day>
     </datetime>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>2022</year>
      <month>12</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1900</year>
      <month>9</month>
      <day>14</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy HH:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="startDateTime">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>2</month>
      <day>7</day>
     </datetime>
    </property>
    <property name="date">
     <date>
      <year>2022</year>
      <month>2</month>
      <day>7</day>
     </date>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>2022</year>
      <month>12</month>
      <day>30</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1900</year>
      <month>9</month>
      <day>14</day>
     </datetime>
    </property>
    <property name="maximumDate">
     <date>
      <year>2022</year>
      <month>12</month>
      <day>30</day>
     </date>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy HH:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Select date and time&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:400;&quot;&gt;Start:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:400;&quot;&gt;End:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="parameter1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Pmp</string>
    </property>
    <item>
     <property name="text">
      <string>Pmp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vmp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Imp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Isc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TempMod</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TempAmb</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Filters&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="parameter1_max">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Max</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="parameter2_max">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Max</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="parameter3_max">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Max</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="parameter4_max">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>540</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Max</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="parameter2_min">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Min</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="parameter3_min">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>480</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Min</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="parameter1_min">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Min</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="parameter4_min">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Min</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="parameter2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Pmp</string>
    </property>
    <item>
     <property name="text">
      <string>Pmp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vmp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Imp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Isc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TempMod</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TempAmb</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="parameter3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Pmp</string>
    </property>
    <item>
     <property name="text">
      <string>Pmp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vmp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Imp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Isc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TempMod</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TempAmb</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="parameter4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>540</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Pmp</string>
    </property>
    <item>
     <property name="text">
      <string>Pmp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vmp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Imp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Isc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Voc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TempMod</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TempAmb</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="visualizeButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>610</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualize</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
